File: app/src/main/java/co/tpcreative/supersafe/common/views/FixedAppBarLayoutBehavior.java
package co.tpcreative.supersafe.common.views;
import android.content.Context;
import android.util.AttributeSet;
import androidx.annotation.NonNull;
import com.google.android.material.appbar.AppBarLayout;

public class FixedAppBarLayoutBehavior extends AppBarLayout.Behavior {

    public FixedAppBarLayoutBehavior(Context context, AttributeSet attrs) {
        super(context, attrs);
        setDragCallback(new DragCallback() {
            @Override public boolean canDrag(@NonNull AppBarLayout appBarLayout) {
                return false;
            }
        });
    }
}

File: app/src/main/java/co/tpcreative/supersafe/common/views/FixedAppBarLayoutBehavior.kt
package co.tpcreative.supersafe.common.views
import android.content.Context
import android.util.AttributeSet
import com.google.android.material.appbar.AppBarLayout

class FixedAppBarLayoutBehavior(context: Context?, attrs: AttributeSet?) : AppBarLayout.Behavior(context, attrs) {
    init {
        setDragCallback(object : DragCallback() {
            override fun canDrag(appBarLayout: AppBarLayout): Boolean {
                return false
            }
        })
    }
}